src/services/help-modal-service/help-texts/usertask-usage.ts

```diff
@@ -0,0 +1,43 @@
+import {HelpText} from '../../../contracts/index';
+
+export const UserTaskUsage: HelpText = {
+  title: 'How to use UserTasks',
+  message: `To use UserTasks you must add one or more form fields to the task.
+
+    The form fields have different types such as:
+
+    - Text (shows an input field for texts)
+    - Integer (shows an input field for numbers)
+    - Decimal (shows an input field for decimal number)
+    - Truth value (shows a checkbox, especially if the UserTask is a "confirm" control)
+    - Date (shows an input field for dates)
+    - Enumeration (shows a dropdown list)
+    - Custom
+
+    Each form field also has the following properties:
+
+    - ID (the ID of the form field used for the token)
+    - Label
+    - Default Value
+
+    The "label" and the "default value" can be set via the process token.
```

Comment:
Das klingt so, als _muss_ man den token benutzen, um Werte für label und default value zu setzen. 
Aber das muss man ja garnicht.

```suggestion
    The values for "label" and "default value" can be anything. 
    However, you can also use token expressions.
```